packages/design-system/src/components/Badge/Badge.tsx

```diff
@@ -1,35 +1,48 @@
-import { css, styled } from 'styled-components';
+import { css, styled, DefaultTheme } from 'styled-components';
 
 import { Flex, FlexComponent, FlexProps } from '../../primitives/Flex';
 import { Typography } from '../../primitives/Typography';
 import { DefaultThemeOrCSSProp } from '../../types';
 
 type BadgeSize = 'S' | 'M';
 
+type BadgeVariant = 'success' | 'primary' | 'danger' | 'warning' | 'neutral' | 'secondary' | 'alternative';
+
 interface BadgeProps extends FlexProps {
   /**
    * If `true`, it changes the `backgroundColor` to `primary200` and the `textColor` to `primary600`
    */
   active?: boolean;
+
   backgroundColor?: DefaultThemeOrCSSProp<'colors', 'background'>;
   /**
    * @default 'M'
    */
   size?: BadgeSize;
+
   textColor?: DefaultThemeOrCSSProp<'colors', 'color'>;
+
+  /**
+   * @default 'neutral'
+   */
+  variant?: BadgeVariant;
 }
 
 const Badge = ({
   active = false,
   size = 'M',
-  textColor = 'neutral600',
-  backgroundColor = 'neutral150',
+  textColor,
+  backgroundColor,
```

Comment:
Here you are deleting the default behavior which will change the default Badges everywhere we use it in the app codebases
```suggestion
  textColor = 'neutral600',
  backgroundColor = 'neutral150',
```